client/test/journal_client_test.ts

```diff
@@ -135,8 +135,10 @@ snapshotTest("JournalClient.read content test", async ({ assertSnapshot }) => {
   const docStream = parseJournalDocuments(stream!);
   const results = await readStreamToEnd(docStream);
 
+  let filtered_results = results.filter(r=>!(r._meta as any).ack)
```

Comment:
This is purely speculative, so take it with a grain of salt. But I think this might still be a flappy test because the number of documents in the journal isn't deterministic. If that's right, then perhaps we just limit this to the first `n` (10?) documents, so we're basically just relying on the hello-world connector to produce _at least_ `n`, instead of _exactly_ `n`.

Reply:
Mmhmmmmmm. I'll make that change